dags/readme.md

```diff
@@ -1,20 +1,116 @@
+# Airflow
 
-**Only put DAGs for data intake in this folder,** DAGs for data processing related to projects should be in their respective project repositories.
+We use [Airflow](airflow.apache.org/) to schedule our pipelines for data we can get from external data providers, and for scheduling internal aggregations. This folder contains ["DAGs"](https://airflow.apache.org/docs/stable/tutorial.html#it-s-a-dag-definition-file) which are Python files that specify the data pipeline scripts to run, the schedule for them to run on, the order of tasks, and what to do if something fails for each pipeline. 
 
 Consult our internal documentation for more tips and resources for writing DAGs.
 
-## Deploying an Airflow Script on Our EC2
+## DAG Naming
+Our preferred naming convention for DAGs is **`dataset`_`verb`**. The .py file should also ideally match the DAG name.  
 
-Our instance of Airflow runs as its own user `airflow` which resides in `/etc/airflow`. In order to ensure the code running on Airflow to be in sync with the code in GitHub, we have developed a unique folder structure. The important folders are:
+You may notice that many older DAGs have not been renamed to this standard: it is usually done as part of a major refactor since renaming the DAG makes the history inaccessible from the UI. 
 
-- `/etc/airflow/dags`: for DAGs. In order for them to be synced to their files in `git`, it is preferable to symbolically link from this folder to the respective `dags` folder in the following folder. For ex: while in `/etc/airflow/dags` run `ln -sf /etc/airflow/data_scripts/dags/pull_here.py .`
-- `/etc/airflow/data_scripts`: a copy of this repository at the `master` branch
+## Contents of DAGs Folder
+**Only put DAGs for data intake in this folder,** DAGs for data processing related to projects should be in their respective project repositories.
+
+### Weather
+- **[weather_pull.py](weather_pull.py)**: [readme](../weather/README.md#data-pipeline---weather_pull-dag).  
+- Deprecated: [pull_weather.py](pull_weather.py)
+
+### Misc
+- [**log_cleanup.py**](log_cleanup.py): A maintenance workflow to periodically clean out the Airflow task logs to avoid those getting too big.
+- [eoy_create_tables.py](eoy_create_tables.py): An end of year maintenance DAG which creates new partitions and maintains calendar tables.  
+
+### Here
+- **[pull_here.py](pull_here.py)**: [readme](../here/traffic/README.md#probe_path).  
+- **[pull_here_path.py](pull_here_path.py)**: [readme](../here/traffic/README.md#path).  
+- HERE Aggregations: [citywide_tti_aggregate.py](citywide_tti_aggregate.py), [traffic_transfer.py](traffic_transfer.py).
```

Comment:
traffic_transfer is not a here aggregation, seems like its doing some transfer stuff in the `traffic` schema?

Reply:
Ah yes it looks like an outdated part of the replication pipeline.